enarx-keep/src/backend/kvm.rs

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::backend::{self, Datum, Keep};
+use crate::binary::Component;
+
+use kvm_ioctls::Kvm;
+
+use std::fs::File;
+use std::io::Result;
+use std::path::PathBuf;
+use std::sync::{Arc, Mutex};
+
+fn dev_kvm() -> Datum {
+    let version = Kvm::new().and_then(|kvm| Ok(kvm.get_api_version()));
+    let (pass, info) = match version {
+        Ok(v) => (v == 12, Some("/dev/kvm".into())),
```

Comment:
Can we split this datum into two data: one for `/dev/kvm` and one for the kvm version? This can help with debugging later.